### PR TITLE
Add autocomplete entries for about pages

### LINKED ISSUE #2252: Add autocomplete entries for about pages
Add another autocomplete provider for about pages.

### PR DESCRIPTION
**Pull request by darkdh**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

Fix #2252

### CODE REVIEW

File: js/components/urlBarSuggestions.js [hunk 1]
```diff
@@ -144,6 +145,7 @@ class UrlBarSuggestions extends ImmutableComponent {
     }
     addToItems(bookmarkSuggestions, 'bookmarksTitle', locale.translation('bookmarksSuggestionTitle'), 'fa-star-o')
     addToItems(historySuggestions, 'historyTitle', locale.translation('historySuggestionTitle'), 'fa-clock-o')
+    addToItems(aboutPagesSuggestions, 'aboutPagesTitle', 'Brave pages', null)
```
Comment (bbondy) [inline js/components/urlBarSuggestions.js]: Brave pages here needs to be taken from `locale.translation` pls

File: js/constants/config.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ module.exports = {
     maxBookmarkSites: 2,
     maxHistorySites: 2,
     maxSearch: 3,
-    maxTopSites: 5
+    maxTopSites: 5,
+    maxAboutPages: 5
```
Comment (bbondy) [inline js/constants/config.js]: let's make this 2

**Comment by bbondy** [comment]
just a couple of nits, otherwise looks great!

**Comment by darkdh** [comment]
Hi @bbondy ,

I made some changes according to your feedback, please take a look at it. Thank you.

**Comment by bbondy** [comment]
Sorry I should have mentioned one more thing that you should only add the localization to the english locale which is the main one.   The other files get auto generated from transifex.com.   Everything else looks good though.

**Comment by darkdh** [comment]
@bbondy Thanks for the info and here is my new patch.

**Comment by bbondy** [comment]
looks great, thank you!

**Pull request merged**